File: Ruby/easy/easy2/odd_numbers.rb
# Odd Numbers

=begin
  
# Print all off numbers (1..99)
# All #'s printed on separate lines

(1..99).each do |num|
  puts num if num.odd?
end
  
=end


# Further Exploration

1.upto(99) do |num|
  puts num if num.odd?
end

File: javascript/js-101-small-problems/easy1/odd-numbers.js
// Log all odd numbers from 1 to 99,
// inclusive, to the console. Log all numbers on separate lines.

let counter = 1;

while (counter <= 99) {
  if (counter % 2 == 1) {
    console.log(counter);
  }
  counter++;
}

console.log('done!');
